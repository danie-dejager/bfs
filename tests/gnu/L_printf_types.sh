bfs_diff -L links -printf '(%p) (%l) %y %Y\n'
