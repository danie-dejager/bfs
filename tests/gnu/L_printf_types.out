(links) () d d
(links/broken) (nowhere) l N
(links/deeply) () d d
(links/deeply/nested) () d d
(links/deeply/nested/broken) (nowhere) l N
(links/deeply/nested/dir) () d d
(links/deeply/nested/file) () f f
(links/deeply/nested/link) () f f
(links/file) () f f
(links/hardlink) () f f
(links/notdir) (symlink/file) l N
(links/skip) () d d
(links/skip/broken) (nowhere) l N
(links/skip/dir) () d d
(links/skip/file) () f f
(links/skip/link) () f f
(links/symlink) () f f
